ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini

```diff
@@ -85,8 +85,9 @@ ckan.auth.user_create_organizations = false
 ckan.auth.user_delete_groups = false
 ckan.auth.user_delete_organizations = false
 ckan.auth.create_user_via_api = false
-ckan.auth.create_user_via_web = true
+ckan.auth.create_user_via_web = false
```

Comment:
@FuhuXia can you confirm with the saml extension this is the correct setting? Or do we want something else?

Reply:
As of now, saml2 extension is using a separated production.ini template. So any time we make changes to production.ini file, the same changes need to be applied to ansible/roles/software/ckan/inventory/templates/inventory-next/etc_ckan_production.ini.j2

Reply:
So, we should update inventory-next/etc_ckan_production.ini.j2 in this PR?

Reply:
Yes, although I doubt this will have any effect on saml login behaviour. But for consistency, we need to keep two production.j2 synchronized.

Reply:
Updated ansible/roles/software/ckan/inventory/templates/inventory-next/etc_ckan_production.ini.j2 with the changes.